src/View/OpenPatientProgWindow.py

```diff
@@ -0,0 +1,82 @@
+import platform
```

Comment:
OpenPatientProgressWindow? everywhere else uses progress though

Reply:
This class inherits from the generic `ProgressWindow` class. This is because the `OpenPatientProgWindow` class has functions specific to the progress window that is created when opening a patient. Or so I would assume! I didn't work on this specific part of the code. The decision-making process behind this is only known to @LWMcClymont

Reply:
Yea, I only suggest to rename this class to `OpenPatientProgressWindow` to improve readability

Reply:
Ah! Okay, I get it. My bad. This change will also be made.

Reply:
Change has been made and added to the PR.